app/uk/gov/hmrc/gform/keystore/RepeatingComponentService.scala

```diff
@@ -85,13 +84,14 @@ object RepeatingComponentService {
       }
 
     section.copy(
-      title = buildText(Some(section.title), index, data).getOrElse(""),
+      title = buildText(Some(section.title), index, data).get,
```

Comment:
`.get` is code smell. This is pretty nice opportunity to learn what does it mean to _lift a function_ which you may hear people to talk about.
It goes like this. Let's write function
```
def buildText(ls: LocalisedString, index: Int, data: FormDataRecalculated)(implicit l: LangADT): LocalisedString { ... }
```
And let's than implement function 
```
def buildTextOpt(ls: Option[LocalisedString], index: Int, data: FormDataRecalculated)(implicit l: LangADT): Option[LocalisedString] = { ... }
```
in terms of `buildText` function.